content/en/blog/_posts/2023-12-20-disabling-in-tree-cloud-provider-goes-beta.md

```diff
@@ -0,0 +1,101 @@
+---
+layout: blog
+title: "Removing In-Tree Cloud Providers Graduates to Beta Status"
+date: 2023-12-20
+slug: removing-in-tree-cloud-providers-beta-status
+---
+
+**Authors:** Michael McCune, Andrew Sy Kim
+
+In 2018, the Kubernetes community agreed to form the Cloud Provider Special Interest Group (SIG), with a mission to externalize all cloud provider integrations and remove all the existing in-tree cloud provider integrations. In January 2019, the Kubernetes community approved the initial draft of [KEP-2395: Removing In-Tree Cloud Provider Code][kep2395]. This KEP defines a process by which we can remove cloud provider specific code from the core Kubernetes source tree. From the KEP:
+
+> Motiviation behind this effort is to allow cloud providers to develop and make releases independent from the core Kubernetes release cycle. The de-coupling of cloud provider code allows for separation of concern between "Kubernetes core" and the cloud providers within the ecosystem. In addition, this ensures all cloud providers in the ecosystem are integrating with Kubernetes in a consistent and extendable way.
+
+After many years of development and collaboration across many contributors, SIG Cloud Provider is happy to announce that as of the Kubernetes 1.29 release, [KEP-2395][kep2395] will graduate to Beta status. The graduation to Beta is a big milestone as we’ve had to work through the complexities that come with migrating a core component in a distributed system like Kubernetes. With the graduation to Beta, all Kubernetes core components will have cloud provider integrations disabled by default. This brings us one step closer to its final removal which will happen when the KEP graduates to GA.
```

Comment:
Whilst this is true, please consider writing this from the _reader_'s perspective. The typical reader cares about having to intervene to keep their cluster running much more than their level of interest in the organization layout (names of SIGs, etc).

Reply:
I was thinking about the paragraph as a whole.

Things readers might not care about very much:
- the removal was led by SIG Cloud Provider
- there is a KEP
- the ID number of that KEP
- the feature stage (beta) for that KEP
- how complex the work has been

What they really want to know is: what action they need to take, how soon, and what happens if they don't act.

Reply:
i changed the paragraph to focus more on what you suggested.